docs/configuration.md

```diff
@@ -279,9 +279,16 @@ In the runtime the image names are calculated as `my.own.repo/calico/kube-contro
 
 ### `spec.extensions.storage`
 
-`spec.extensions.storage` controls bundled storage.
+`spec.extensions.storage` controls bundled storage provider.
 The default value `external` makes no storage deployed.
-Value `openebs_local_storage` deploys OpenEBS with the local path set up.
+
+To enable to [embedded host-local storage provider](storage.md#bundled-openebs-storage) use the following configuration:
```

Comment:
Suggestion:
"To enable to [embedded host-local storage provider]"
->
"To enable [OpenEBS local storage]"

Reply:
fixed